docs/tutorial/adding_widgets_to_a_window.md

```diff
@@ -1 +1,93 @@
-To add widgets to the window, we need to create the widgets and set their properties. In this case, we will add a layout to the window and then add a button and a label to the layout. 
+To add widgets to the window, we need to create the widgets and set their properties. In this case, we will first add a central widget to the window, then add a layout to the central widget, and finally add a button and a label to the layout.
+
+<hr>
+
+## Adding a layout
+
+First we need to keep in mind that every [`QMainWindow`](../../QtWidgets/QMainWindow) has to have a central widget. This can simply be a [`QWidget`](../../QtWidgets/QWidget) object. Next, we will add a layout to the central widget. A layout is a container that manages the position of child widgets. In this case, we will use a [`QVBoxLayout`](../../QtWidgets/QVBoxLayout) to manage the position of widgets in our window. The newly added code is highlighted below:
```

Comment:
Maybe we also want to explain in this sentence that the V is from 'vertical layout', e.g. .. use a QVBoxLayout to place the widgets in a vertical layout in our window.